### PR TITLE
Add french colors

### PR DESCRIPTION
**Pull request by sblondon**
The patch allows selection of french color names:

>>> from faker import Factory
>>> fake = Factory.create('fr_FR')
>>> fake.safe_color_name()
'bordeaux'
>>> fake.color_name()
'Rose Passion'

i didn't any issue about adding french colors.

### CODE REVIEW

File: faker/providers/color/fr_FR/__init__.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from collections import OrderedDict
+
+from .. import Provider as ColorProvider
+
+localized = True
```
Comment (fcurella) [inline faker/providers/color/fr_FR/__init__.py]: the `localized` module attribute is not needed anymore. You can delete this line

**Comment by sblondon** [comment]
The 'localized = True' exist in faker/providers/color/ru_RU/__init__.py too.
Do you want another pull request where I remove it? If yes, do you want it on this thread or on another one, with his own branch?

**Comment by fcurella** [comment]
@sblondon I take that back. I tried to remove the `localized` attribute, but I had to revert that change in e5d75edf3786814 because people started writing non-localized 3rd party providers.

I'll merge this in and re-add `localized = True`.

Thank you so much for your contribution! ✨

**Comment by sblondon** [comment]
Thanks you for the quick merge! :-)

**Pull request merged**